website/docs/getting-started/installation.mdx

```diff
@@ -20,42 +20,97 @@ import TOCInline from '@theme/TOCInline';
+  xattr -d com.apple.quarantine ./caesar
+  ```
+
+  </TabItem>
+  <TabItem value="debian" label="Debian/Linux">
+    No additional steps necessary.
+  </TabItem>
+  <TabItem value="win" label="Windows">
+    No additional steps necessary.
+  </TabItem>
+</Tabs>
+:::
+
+### b) Docker Image
+
+We also provide a Docker image.
+
+```bash
+docker pull ghcr.io/moves-rwth/caesar:latest
+```
+
+You can use this Docker image just like the `caesar` binary.
+To access files from your system you need to mount them as volumes.
+For example, if you want to verify a program in `./programs/example.heyvl`, the following command mounts the `./programs/` directory as `/root/caesar/programs/` inside the container and uses `caesar` to verify `example.heyvl`.
+
+```bash
+docker run -it -v $PWD/programs:/root/caesar/programs/ ghcr.io/moves-rwth/caesar:latest /root/caesar/programs/example.heyvl
+```
+
+## Option B: Compiling Locally
+
+### a) Build Binary
+
+#### Installing Dependencies
 
 You will need to install some dependencies manually:
 
 **Rust:** To install [Rust](https://www.rust-lang.org/), we recommend [installation via rustup](https://rustup.rs/).
 You might need to restart your terminal after installing Rust.
 
-**Fish Shell**: We write some scripts written in [fish](https://fishshell.com/).
-
-**Build Tools:** Caesar builds [Z3](https://github.com/Z3Prover/z3) itself, so we need Python and [cmake](https://cmake.org/).
+**Build Tools:** Caesar builds [Z3](https://github.com/Z3Prover/z3) itself, so we need [Python](https://www.python.org) and [cmake](https://cmake.org/).
 
-```mdx-code-block
 <Tabs groupId="operating-systems">
   <TabItem value="mac" label="macOS">
-    <CodeBlock language="bash">{`# Install Rust via rustup
-curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
-# Install CMake and fish
-brew install cmake fish`}
-    </CodeBlock>
+
+  ```bash
+  # Install Rust via rustup
+  curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+  # Install Python and CMake
+  brew install python cmake
+  ```
+
   </TabItem>
   <TabItem value="debian" label="Debian/Linux">
-    <CodeBlock language="bash">{`# Install Rust via rustup
-curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
-# Install CMake, LLVM, clang, and fish
-apt-get install cmake llvm-dev libclang-dev clang fish`}
-    </CodeBlock>
+
+  ```bash
+  # Install Rust via rustup
+  curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+  # Install Python, CMake, LLVM and clang
+  apt-get install python3 cmake llvm-dev libclang-dev clang
+  ```
+
   </TabItem>
   <TabItem value="win" label="Windows">
-    We do not have build instructions for Windows at the moment. We recommend that you use <Link to="https://learn.microsoft.com/en-us/windows/wsl/install">WSL</Link> and refer to the Debian/Linux instructions.
-  </TabItem>
+
+  1. [Follow the instructions](https://rustup.rs/) to install Rust via rustup.
+  2. Install Python, CMake and LLVM
```

Comment:
Here, a link to the Chocolatey install website would be helpful.

Reply:
I disagree. Chocolatey is *the* package manager for Windows (winget does not seem to be as wide-spread yet), just like Homebrew is *the* package manager for macOS.

As these instructions are part of the DIY section, I find it reasonable to expect at least some prior familiarity with the de-facto standard tools on your platform of choice. Otherwise you would just use the default binaries.

(On a similar note: We also don't show how to install Docker, and rightfully so, as I do not think that this is in scope.)